Tests/FunctionalTests/Runner/Main.js

```diff
@@ -12,6 +12,12 @@ let startTime = null;
 
 let perfMode = process.argv.indexOf("-perfmode") !== -1;
 
+const skipAsyncHooksTests = () => {
```

Comment:
Skip "azure sdk" tests on versions of node where azure SDK won't work (must be node 8+, but not 8.0.0)